src/System.ComponentModel.TypeConverter/ref/4.1/System.ComponentModel.TypeConverter.depproj

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We shouldn't no longer need any depproj's. Is there a particular reason you added this?